src/js/dataMapping/contracts/transactionTable.js

```diff
@@ -5,33 +5,38 @@
 
 const tableSearchFields = {
     columnWidths: {
-        modification: 230,
-        action_date: 200,
-        amount: 220,
-        description: 380
+        modification: 184,
```

Comment:
You can test this by replacing the `$font-sans` value in `_variables.scss` to just `$helvetica`:
![screen shot 2017-05-02 at 3 18 02 pm](https://cloud.githubusercontent.com/assets/3594363/25634991/99bbf106-2f4a-11e7-801f-79a40790ac0c.png)